contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -429,9 +429,8 @@ class CaptureRepeatedly(E2ETestCase):
     The test does not verify any behavior, but it will fail if Orbit crashes.
     """
 
-    def _stop_capture_if_necessary(self):
+    def _stop_capture_if_necessary(self, capture_tab):
         logging.info('Querying if a capture is still running')
-        capture_tab = self.find_control('Group', "CaptureTab")
         capture_options_button = self.find_control('Button', 'Capture Options', parent=capture_tab)
```

Comment:
You mentioned that the slow part is to get the tab, right? So getting this button works fine?

Please ignore if you have checked this ...

Reply:
Yes, I tested instrument_functions and capture_repeatedly. At least locally both of them work as expected (run without errors and also doesn't need too much time to stop a capture).